### ISSUE TITLE
desktop app not used olter port

### ISSUE THREAD

**Issue by Vincenzoferrara** [go-vikunja/vikunja#335]
### Description

I installed the desktop app in manjaro (arch)

my server is stable debian, and i have caddy reverse proxy on http and httpsa port (since it provides acme)

but the desktop software tries to connect to the standard port 3456, instead of my 443, even if explicitly declared, using 'https://todo-list.local:443' ((it's my internal domain name, and it works from web)

I would like vikunja to try to look for the standard https port too, like jellyfin does

moreover when you connect you don't know if it has started, as there is no output

### Vikunja Version

my server (Vikunja api version v0.24.4 Built with go1.23.1)

### Can you reproduce the bug on the Vikunja demo site?

Please select

**Comment by kolaente** [comment]
Which desktop version are you using?

**Comment by Vincenzoferrara** [comment]
in aur 24.2.1 link [https://aur.archlinux.org/packages/vikunja-desktop-bin](url)

**Comment by kolaente** [comment]
Does it work if you use one of Vikunja's built packages? https://dl.vikunja.io/desktop/0.24.4

**Comment by Vincenzoferrara** [comment]
I just downloaded the app image , and I have the same problem

**Comment by kolaente** [comment]
Are you sure it is the port that's the issue here? I was able to connect to a Vikunja instance running on a non-standard port without problems.

Did you [enable cors](https://vikunja.io/docs/desktop-packages#installation) on your instance?

**Comment by Vincenzoferrara** [comment]
yes it worked, I read all the documentation before writing but that part is missing sorry for the inconvenience.

I would like to add 1 detail that I noticed

- I would like to add that if you do not specify https vikunja does not try to connect (it would be useful if it was added automatically if it is not specified)

**Comment by kolaente** [comment]
No worries, I've added the specific part on the page I linked to after your issue. It was mentioned elsewhere before, but rather hard to find.

> I would like to add that if you do not specify https vikunja does not try to connect (it would be useful if it was added automatically if it is not specified)

The problem here is that browsers won't allow connecting between different protocols, and thus this won't work in most cases - except the electron app. Will take a look if there's a way to solve this.

Closing this as solved now, since the original issue is solved, please ping or open a new issue if you have any other problems.
